pkg/agent/cniserver/sriov.go

```diff
@@ -0,0 +1,130 @@
+package cniserver
```

Comment:
there is no LICENSE / Copyright header for this file

Reply:
Done